### ISSUE TITLE
`showPagination` doesn't seem to be working

### ISSUE THREAD

**Issue by justin-calleja** [jdalrymple/gitbeaker#759]
**Description**

When using `showPagination` in a call like this:

```js
  const result = await api.Projects.all({ maxPages: 4, perPage: 50, showPagination: true });
```

Documentation says to expect a result of the form:

```
data: [
...
],
pagination: {
  total: 20,
  next: 4,
  current: 2,
  previous: 1,
  perPage: 3,
  totalPages: 3,
}
```

However, I only get the `data` part i.e. an array of objects  of the form:

```json
  id: 123,
  description: '',
   name: '',
...
```

If I go to `/help` on the gitlab instance I'm running against, I get back:

```
GitLab Community Edition 11.11.3
```

(I'm using gitbeaker version `19.3.0`).

Thank you

**Comment by jdalrymple** [comment]
Thats my bad! I must of missed an updated to the docs. Use `showExpanded: true` instead

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`19.3.1`](https://github.com/jdalrymple/gitbeaker/releases/tag/19.3.1) :rocket:
